src/main/java/com/conveyal/gtfs/util/GeoJsonUtil.java

```diff
@@ -192,10 +193,13 @@ private static List<LocationShape> unpackLocationShapes(
         List<String> errors
     ) {
         ArrayList<LocationShape> locationShapes = new ArrayList<>();
+        Set<String> uniqueLocationIds = new HashSet<String>();
```

Comment:
Can we rename this? Maybe something like
```suggestion
        Set<String> seenLocationIds = new HashSet<String>();
```

Reply:
I don't see why not. Please enjoy https://github.com/conveyal/gtfs-lib/pull/374/commits/e2c2444a4cc631d05f017918f8da67fa5cabfaa9